### ISSUE TITLE
Halting the beta version of the Vagrant box renders the box unusable

### ISSUE THREAD

**Issue by tlxo** [Seravo/wp-vagrant#51]
It's not possible to restart Vagrant after a `vagrant halt` with the beta version of the box (a box run with `config.vm.box = 'seravo/wordpress-beta'` in Vagrantfile).

```
==> seravocom-box: Running triggers after up...
checking for composer... no
==> seravocom-box: Executing remote command "composer install --working-dir=/data/wordpress"...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
==> seravocom-box: Waiting for container SSH...
```
There is a workaround: running `vagrant destroy; vagrant up` will clean up whatever went wrong with the `vagrant halt`, but it might still be worth it to investigate this.

**Comment by l3ku** [comment]
I can also reproduce this issue with the `seravo/wordpress-beta (virtualbox, 20180522.0.3)` box by running
```
$ vagrant up
...
$ vagrant halt
...
$ vagrant up
...
==> wordpress-box: Waiting for machine to boot. This may take a few minutes...
    wordpress-box: SSH address: 127.0.0.1:2222
    wordpress-box: SSH username: vagrant
    wordpress-box: SSH auth method: private key
    wordpress-box: Warning: Authentication failure. Retrying...
    wordpress-box: Warning: Authentication failure. Retrying...
    wordpress-box: Warning: Authentication failure. Retrying...
```

**Comment by tlxo** [comment]
Fixed in version 20180528.0.1.
